build/ansible/roles/supervisord-init/tasks/main.yml

```diff
@@ -1,4 +1,8 @@
 ---
+- name: Set fact if running inside Docker
+  set_fact:
+    running_in_docker: "{{ lookup('env', 'RUNNING_IN_DOCKER') | default(false) }}"
```

Comment:
couldn't we pass this fact as variable like we do for pmm_server_image in v3 or variables here https://github.com/percona/pmm/blob/221c484e94760d4cb24ba9575062e3a7908c7a6c/build/ansible/roles/pmm2-images/vars/main.yml#L4 ?
I think it would be more clear code, don't you think so?

Reply:
I've made the changes accordingly. Now passing Ansible variable instead of setting environment variable inside Dockerfile.el9